spec/smoke/api/v1/providers_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper_smoke"
+
+describe "V1 Public API Smoke Tests", :aggregate_failures, smoke: true do
+  let(:recruitment_year) { Settings.current_recruitment_cycle_year }
+  let(:base_url) { Settings.publish_url.sub("www", "api") }
+
+  subject(:response) { HTTParty.get(url) }
+
+  let(:payload) { JSON(response.body).fetch("data", []) }
+
+  context "providers" do
```

Comment:
@AbigailMcP Should your one has context as well?

Reply:
I'll add!